packages/palette/src/Theme.tsx

```diff
@@ -49,6 +49,12 @@ export const useTheme = <T extends TTheme>() => {
   return { theme }
 }
 
+/** Returns a config specific to the current theme */
+export const useThemeConfig = <T, U>({ v2, v3 }: { v2: T; v3: U }) => {
+  const { theme = { id: "v2" } } = useTheme()
```

Comment:
Setting a default here now so in most cases you won't have to wrap in a theme. The only reason `Button` requires it right now is because of that hover media query. Added a `mountWithTheme` function as well anyway.

Reply:
> The only reason Button requires it right now is because of that hover media query.

Maybe we can make it so this isn't required? Can we skip the mq and do it a different way?

Reply:
I actually just am PRing that separately now because it's gonna cause problems with Force specs 😬